lib/config-migrator.js

```diff
@@ -0,0 +1,59 @@
+/** @babel */
+
+import _ from 'lodash'
+import { heredoc } from './werkzeug'
+
+export default async function checkConfigAndMigrate () {
```

Comment:
Can we avoid having this be async? I'm just worried since we're calling it in a _fire and forget_ manner from the main module, and that introduces the possibility of a race condition. Either we make it sync (and wait for the async call to `latex.builderRegistry.checkRuntimeDependencies()`), or we have to wait in the main module.